src/CommunityToolkit/Xamarin.CommunityToolkit/ObjectModel/Extensions/ObservableRangeCollectionEx.shared.cs

```diff
@@ -0,0 +1,17 @@
+﻿using System.Collections.Generic;
+using System.ComponentModel;
+
+namespace Xamarin.CommunityToolkit.ObjectModel
+{
+	public static class ObservableRangeCollectionEx
```

Comment:
```suggestion
	public static class ObservableRangeCollectionExtensions
```

Reply:
@pictos are you strictly against adding this method directly to ObservableRangeCollection?

Reply:
Yes, because this a _hack_ (I'm not saying that's evil or bad) so I prefer to have it outside the `ObservableRangeCollection` implementation.